internal/gcsx/bucket_manager.go

```diff
@@ -73,18 +76,20 @@ type BucketManager interface {
 }
 
 type bucketManager struct {
-	config BucketConfig
-	conn   *Connection
+	config        BucketConfig
+	conn          *Connection
+	storageHandle storage.StorageHandle
```

Comment:
Please make storage.StorageHandle as pointer.

Reply:
checked the code
connection is a struct and storageHandle is a Interface. Since interface is kind of pointer which holds the concrete instance no need to make it as pointer.

Reply:
Method is not callable through pointer